x-pack/plugins/security_solution/cypress/tasks/timeline.ts

```diff
@@ -205,7 +207,7 @@ export const openTimelineTemplateFromSettings = (id: string) => {
 };
 
 export const openTimelineById = (timelineId: string) => {
-  return cy.get(TIMELINE_TITLE_BY_ID(timelineId)).click({ force: true });
+  return cy.get(TIMELINE_TITLE_BY_ID(timelineId)).pipe(($el) => $el.trigger('click'));
```

Comment:
You don't have a condition right after this such as checking if something is visible or not visible or another condition after the pipe. Might be why it's still failing. Once that `$el` returns I believe it is going to think everything is good.